<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="main_stack">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="profile">
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>420</width>
        <height>341</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="first_name">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>80</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="last_name">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>110</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>First Name</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>110</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Last Name</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>140</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Weight, kg</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>170</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Height, cm</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>200</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Birthday(DD/MM/YYYY)</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="weight">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>140</y>
         <width>111</width>
         <height>26</height>
        </rect>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="height">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>170</y>
         <width>111</width>
         <height>26</height>
        </rect>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
      <widget class="QDateEdit" name="birthday">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>200</y>
         <width>110</width>
         <height>26</height>
        </rect>
       </property>
       <property name="displayFormat">
        <string>dd/MM/yyyy</string>
       </property>
      </widget>
      <widget class="QPushButton" name="submit_button">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>310</y>
         <width>83</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="main">
     <widget class="QPushButton" name="new_scan_button">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>210</y>
        <width>83</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>New Scan</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>371</width>
        <height>51</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QComboBox" name="switch_profile">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>131</width>
         <height>25</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>New Item</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="edit_profile">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>83</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
      <widget class="QPushButton" name="delete_profile">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>20</y>
         <width>83</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>62</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Profiles</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="results">
     <widget class="QTabWidget" name="results_tabs">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>581</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="diagram">
       <attribute name="title">
        <string>diagram</string>
       </attribute>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>781</width>
          <height>511</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="indicators">
       <attribute name="title">
        <string>indicators</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="recommendations">
       <attribute name="title">
        <string>recommendations</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="scan"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
